src/App.js

```diff
@@ -26,16 +34,43 @@ class SimpleBox extends Component {
           this.setState(() => {
             return { clicked: !this.state.clicked };
           })}
+        color={this.props.color}
       />
     );
   }
 }
 
+class ColumnOfBoxes extends Component {
+  render() {
+    return (
+      <div className="columnOfBoxes">
+        <SimpleBox color={this.props.color} />
```

Comment:
you can declare a color in a top like `const {color} = this.props`

Reply:
Also maybe you can use a loop and generate this dynamic instead of copy pasting.